add_gist.rb

```diff
@@ -0,0 +1,62 @@
+require 'rubygems'
+require 'net/http'
+require 'uri'
+require 'json'
+require 'OpenSSL'
+require 'find'
+require 'dotenv/load'
+
+# Module that gathers the methods necessary to add a new gist
+module AddGist
+  GISTS_URL = 'https://api.github.com/gists'.freeze
+
+  module_function
```

Comment:
did rubocop made you use `module_function`? I much rather have `self.` on every method

Reply:
Initially I used ```extend self``` and rubocop suggested ```module_function```
I now added ```self.``` on every method and it works great